src/clj/runbld/notifications/slack.clj

```diff
@@ -0,0 +1,41 @@
+(ns runbld.notifications.slack
+  (:refer-clojure :exclude [send])
+  (:require [runbld.schema :refer :all]
+            [schema.core :as s]
+            [cheshire.core :as json]
+            [clj-http.client :as http]
+            [runbld.io :as io]
+            [runbld.notifications :as n]
+            [runbld.store :as store]
+            [stencil.core :as mustache]))
+
+(s/defn send :- s/Any
+  [opts :- MainOpts
+   ctx  :- NotifyCtx]
+  (let [f    (-> opts :slack :template)
+        tmpl (-> f io/resolve-resource slurp)
+        color (if (-> ctx :process :failed)
+                "danger"
+                "good")
+        js   (mustache/render-string tmpl (assoc ctx :color color))]
+    (http/post (-> opts :slack :hook)
+               {:body js})))
```

Comment:
I don't know what runbld does when exceptions are thrown, but you may want `:throw-exceptions false` here as well, and maybe `:socket-timeout` and `:conn-timeout` additionally